releasenotes-builder/src/main/java/com/github/checkstyle/globals/ReleaseNotesMessage.java

```diff
@@ -114,6 +114,15 @@ public String getAuthor() {
         return author;
     }
 
+    /**
+     * Sets the title.
+     *
+     * @param newTitle new title.
+     */
+    public void setTitle(String newTitle) {
```

Comment:
Instead of mutation, please make new field - githibEsacedTitle.

Reply:
May be https://freemarker.apache.org/docs/dgui_misc_autoescaping.html

Reply:
Done.